src/skills-builder/skills-builder-modal/select-preferences/SelectPreferences.jsx

```diff
@@ -14,8 +14,8 @@ const SelectPreferences = () => {
   const { currentGoal, currentJobTitle } = state;
 
   return (
-    <>
-      <p className="lead mb-5">
+    <Stack gap={4}>
+      <p className="lead">
         <FormattedMessage {...messages.skillsBuilderDescription} />
```

Comment:
Is this another line where we should have updated the way we're handling i18n?

Reply:
It looks like I missed a few more of these. I'll run through the app and make sure they're all the same